packages/story-editor/src/app/highlights/quickActions/useQuickActions.js

```diff
@@ -217,12 +226,31 @@ export const MediaPicker = ({ render, ...props }) => {
       showSnackbar,
     ]
   );
+
+  const onUpload = useCallback(
+    (resource) => {
+      prependMedia({
+        media: [resource],
+      });
```

Comment:
I am seeing this issue in main. So if a drag a video into canvas when I am filtering image, I see it prepended to media library. I think We should add this to `prependMedia` reducer but in maybe into another PR, as it is another issue,

Reply:
This took a lot longer to track down than I thought. I have a fix here, see what you https://github.com/GoogleForCreators/web-stories-wp/pull/11941/commits/dbfeb95d7c03162512628453214e9916c5a31af5

Reply:
> I am seeing this issue in main. So if a drag a video into canvas when I am filtering image, I see it prepended to media library. 

Right. Just found #9964 because of this, indicating that we did that on purpose.

dbfeb95d7c03162512628453214e9916c5a31af5 means that you'll still see the uploading items when you change the filter again, right?

That couldn't be done in the reducer, so it kinda makes sense to put it there.

On the other hand, it's weird that we have to filter the media elements at that stage. Would be better to put this into `useContextValueProvider` directly, not have it in the component.

But yes, let's do that in a separate PR so we can refine & test it better.